libs/model/src/lib/permissions.ts

```diff
@@ -52,3 +52,9 @@ export function createDocPermissions(params: Partial<DocPermissionsDocument> = {
     ...params
   };
 }
+
+export interface Permissions extends PermissionsDocument {
+  docPermissions?: DocPermissions[];
+}
+
+type DocPermissions = DocPermissionsDocument;
```

Comment:
in fact (my mistake) you should be able to do:

```suggestion
export interface Permissions extends PermissionsDocument {
  docPermissions?: DocPermissionsDocument[];
}
```